katti/localproblems.py

```diff
@@ -201,7 +201,7 @@ def get_random_problem(desired_rating: float, user_conf: dict, problems_conf: di
             print("Invalid rating. Rating must be between 1 and 10")
             print("Aborting...")
             sys.exit(1)
-        choices = {i for i in problems_conf if rating <= problems_conf[i] < (rating + 1)}
+        choices = {i for i in problems_conf if rating <= float(problems_conf[i]) < (rating + 1)}
```

Comment:
Is this strictly required in python? Python seems to correctly compare floats and integers

Reply:
the selected problems got stored as strings in the config file

Reply:
I had to make this change because this function uses the selected and random config. I could regenerate the selected json so it's saved as a float and not a string

Reply:
No, that makes sense. In the old code it was an int, but with the change that's understandable.